lib/translations_checker/locale_file.rb

```diff
@@ -0,0 +1,59 @@
+require "translations_checker/delegation"
+require "translations_checker/locale_file_key_map"
+require "translations_checker/locale_file_content"
+
+require "pathname"
+
+# :reek:TooManyInstanceVariables
+module TranslationsChecker
+  class LocaleFile
+    include Delegation
+
+    attr_reader :path
+
+    delegate :exist?, :to_s, to: :path
+    delegate :key_at,        to: :key_map
+    delegate :[],            to: :content
+
+    def initialize(path)
+      @path = Pathname(path)
```

Comment:
was testing it and this would error if path was a `TranslationChecker::LocaleFile` `to_s` seems to fix